src/DebugChannel.js

```diff
@@ -11,6 +11,8 @@
 import path from "path";
 import type { DebuggerOptions } from "./options";
 
+const TWO_DASH = "--";
```

Comment:
I would call it LENGTH_SEPARATOR which is the real purpose of this const. The fact we are using two dash is an implementation details which may change.